demos/basics/loading-saving-neo4j.ipynb

```diff
@@ -0,0 +1,2038 @@
+{
```

Comment:
<blockquote>and one type of edge that touches it.</blockquote><p>maybe "and one type of edge between them"? or "one type of edge connecting them"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>